addons/mass_mailing_sms/tests/test_sms_mailing_ab_testing.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from datetime import datetime
+
+from odoo.addons.mass_mailing_sms.tests.common import MassSMSCommon
+from odoo.addons.mass_mailing.tests.test_mailing_ab_testing import TestMailingABTesting
+from odoo.tests import users, tagged
+
+
+@tagged('post_install', '-at_install')
+class TestSMSMailingABTesting(MassSMSCommon, TestMailingABTesting):
+    def setUp(self):
+        super().setUp()
+        self.ab_testing_sms_mailing_1 = self.env['mailing.mailing'].create({
+            'subject': 'A/B Testing SMS V1',
+            'contact_list_ids': self.mailing_list.ids,
+            'ab_testing_enabled': True,
+            'ab_testing_pc': 10,
+            'ab_testing_schedule_datetime': datetime.now(),
+            'mailing_type': 'sms'
+        })
+        self.ab_testing_sms_mailing_2 = self.ab_testing_sms_mailing_1.copy({
+            'subject': 'A/B Testing SMS V2',
+            'ab_testing_pc': 20,
+        })
+        self.ab_testing_sms_campaign = self.ab_testing_sms_mailing_1.campaign_id
+        self.ab_testing_sms_mailing_ids = self.ab_testing_sms_mailing_1 + self.ab_testing_sms_mailing_2
+        self.ab_testing_sms_mailing_ids.invalidate_cache()
+
+    @users('user_marketing')
+    def test_ab_testing_sms(self):
```

Comment:
Actually just having the initial test, but in this module, should be sufficient, no need to write it both ways IMO.

That way we also avoid having to create extra data at setup and simplify everything.